includes/model/class-user.php

```diff
@@ -246,73 +247,7 @@ public function get_endpoints() {
 	 */
 	public function get_attachment() {
 		$extra_fields = get_actor_extra_fields( $this->_id );
```

Comment:
should we move the `get_actor_extra_fields()` global function to the collection class, to simplify things? I think after your re-write the global function makes no more sense?!?

Reply:
Yeah that's probably a good idea. Wanted to keep the helper function in place initially but it's not doing much for us any more.